fillingDataBase.py

```diff
@@ -0,0 +1,199 @@
+from main import *
```

Comment:
Wildcard imports (from <module> import *) should be avoided (c) https://www.python.org/dev/peps/pep-0008/#imports